### ISSUE TITLE
multiple versions shown as Latest

### ISSUE THREAD

**Issue by bhuvak** [NuGet/NuGetGallery#2598]
When multiple versions of a package are uploaded in parallel, there is a probability of multiple version being updated in DB at the same time causing IsLatest to be set to true on both of them.

https://www.nuget.org/packages/System.Runtime/

**Comment by maartenba** [comment]
Manually fixed these occurences in database. @bhuvak can you check catalog/search?

We'll discuss a fix for this via email.

**Comment by maartenba** [comment]
Closing - duplicate of #2514
